src/components/Footer.tsx

```diff
@@ -213,20 +201,78 @@ const Logo = styled.div`
 const FooterLinkGroups = styled.div`
 	display: flex;
 	flex: 1;
+	width: 100%;
```

Comment:
alot of `100%` isn't advisable. divs are already block elements, if perharps it's parent has `display:flex` then using the parent's align property is better to achieve a full width

Reply:
Done